stackdriver.go

```diff
@@ -343,9 +343,8 @@ func NewExporter(o Options) (*Exporter, error) {
 
 // ExportView exports to the Stackdriver Monitoring if view data
 // has one or more rows.
-func (e *Exporter) ExportView(vd *view.Data) {
-	e.statsExporter.ExportView(vd)
-}
+// Deprecated: this method does not have any effect now, use ExportMetrics instead.
+func (e *Exporter) ExportView(vd *view.Data) {}
```

Comment:
Remove this, prefer to break build rather than be a no-op :)

Reply:
Make sense. I expect the effect to be minimal since we switched to exporting metrics long ago.